orga/app/components/user-logged-menu.js

```diff
@@ -52,7 +52,9 @@ export default Component.extend({
 
       await this.currentUser.load();
```

Comment:
@MathieuGilet j'avais pas vu passer cette implémentation quand elle a été faite mais quand j'avais plutôt en tête que le changement d'organisation réinitialiserait complètement l'application (`window.location.replace('/')` par exemple).

L'idée était justement de se prémunir d'accidents comme celui sur les `queryParams`, ou d'autres encore inconnus touchant à divers états de l'application associés à l'organisation précédente, et aussi d'éviter la duplication du code de login comme `this.currentUser.load`.